codemagic.yaml

```diff
@@ -0,0 +1,99 @@
+definitions:
+  configure_environment: &configure_environment
+    instance_type: mac_mini_m1
```

Comment:
Why do we need to run this on a mac mini instead of a Linux machine that costs more credits?

Reply:
@luongvo Because it's free. 😂  Ref: https://docs.codemagic.io/knowledge-codemagic/machine-type/

Reply:
> Note: Linux Standard and Premium machines are only available for teams and users that have [billing enabled](https://docs.codemagic.io/billing/billing). See the [pricing page](https://codemagic.io/pricing/) for more information.

Reply:
Okay, Linux and Windows machine seem to be cheaper but require billing enabled. Should we have a comment here to switch to not always use macm1?

Reply:
@luongvo Do you want me to do the document along with this script? 😂

Reply:
@luongvo Added here: 57c72d79